easybuild/tools/filetools.py

```diff
@@ -296,7 +408,7 @@ def find_easyconfigs(path, ignore_dirs=None):
                 continue
 
             spec = os.path.join(dirpath, f)
-            _log.debug("Found easyconfig %s" % spec)
+            _log.debug("Found easyconfig %s", spec)
```

Comment:
the raise issue is not relevant here (and @riccardomurri already fixed the raise issue)